query-engine/js-connectors/js/smoke-test-js/prisma/postgres-neon/schema.prisma

```diff
@@ -33,8 +33,8 @@ model type_test {
   date_column_null      DateTime?                   @db.Date
   time_column           DateTime                    @db.Time(0)
   time_column_null      DateTime?                   @db.Time(0)
-  datetime_column       DateTime
-  datetime_column_null  DateTime?
+  datetime_column       DateTime                    @db.Timestamp(3)
+  datetime_column_null  DateTime?                   @db.Timestamp(3)
```

Comment:
I'm a bit confused 🤔 `datetime_column` looks a lot like `timestamp_column` now (only the `@db.Timestamp()` value differs, 3 vs 0)

We should also test without `@db.Timestamp(3)` like before, maybe?

Reply:
Should this be `@db.DateTime(0)`? 

I found that in https://github.com/prisma/prisma-engines/blob/1f499fcd4a1257371e393196da9a30225513b2a3/query-engine/js-connectors/js/smoke-test-js/prisma/mysql-planetscale/schema.prisma#L44

Reply:
I guess I'm also confused that the changes are about `postgres-neon` and there is nothing about `mysql-planetscale` (the bug was found when using PlanetScale adapter)

Reply:
`@db.DateTime()` doesn't exist on Postgres, whereas MySQL has both `@db.DateTime()` and `@db.Timestamp()`.
While I haven't committed the MySQL schema.prisma, the change should work for both MySQL and Postgres providers.